### ISSUE TITLE
Consider changing naming convention?

### ISSUE THREAD

**Issue by mstahv** [mstahv/v-leaflet#52]
Should the Vaadin API have non prefixed class names and GWT API be G prefixed?

I guess most users are using the Vaadin API, so it should be considered first class citizen. If nobody disagrees, we could make the change in next larger release. To keep backwards compatibility, aliases with L prefix for relevant layers could be kept for a short time.

Opinions?

**Comment by octavm** [comment]
I've created a vaadin addon for V-Leaflet of Leaflet.RotatedMarker plugin by Benjamin Becquet ([(https://github.com/bbecquet/Leaflet.RotatedMarker)](url))
The addon is on my github page ([https://github.com/octavm/v-leaflet-rotatedmarker-parent](url))
Would it be worth it adding it to vaadin directory, maybe it will help other V-Leaflet users ?

**Comment by mstahv** [comment]
Sure! Just build the zip files and upload them to https://vaadin.com/directory

If your face some issues with the process, let me know, maybe I can help.

**Comment by mstahv** [comment]
Congrats, your first Vaadin add-on appears to be out! I think you should also publish a version of the g-leaflet-rotatedmarker module to make it really functional? I didn't try it but just guessed based on the github project structure.

**Comment by octavm** [comment]
Thanks for the suggestion. Done!

**Comment by octavm** [comment]
Regarding your naming convention question, if I may suggest, I think the g-leaflet classes should be named LMarker, LTileLayer, etc to match with Leaflet js API and the v-leaflet classes should just be named Marker, TileLayer, etc.

**Comment by mstahv** [comment]
Agreed, I'll try to do the refactoring at some point before the final 1.0 release. I guess could leave deprecated LMarker/LMap/... (extends Marker) classes still to v-leaflet for backward compatibility, at least for couple of releases. In GWT wrapper we could do the same as well, but it is not that essential there, just affects extensions like your new rotated marker add-on.

Let me know if you want to take care of this refactoring.
